railties/lib/rails/generators/rails/app/templates/config/initializers/callback_terminator.rb

```diff
@@ -1,5 +1,6 @@
 # Be sure to restart your server when you modify this file.
 
-# Do not halt callback chains when a callback returns false. This is a new Rails 5.0 default, 
-# so introduced as a config to ensure apps made with earlier versions of Rails aren't affected when upgrading.
+# Do not halt callback chains when a callback returns false. This is a new
+# Rails 5.0 default, so it is introduced as a config to ensure that apps made
+# on earlier versions of Rails are not affected when upgrading.
```

Comment:
There is the same addition in `active_record_belongs_to_required_by_default.rb`.

This change could be removed from this PR and added to another one for both config files.